game-core/src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -1920,8 +1920,8 @@ private void setCreatesUnitsList(final String value) throws GameParseException {
           "createsUnitsList: No unit called:" + unitTypeToProduce + thisErrorMsg());
     }
     final int n = getInt(s[0]);
-    if (n < 1) {
-      throw new GameParseException("createsUnitsList must have positive values" + thisErrorMsg());
+    if (n < 0) {
+      throw new GameParseException("createsUnitsList cannot have negative values" + thisErrorMsg());
```

Comment:
nit, missing the trailing space before 'thisErrorMsg()'

suggestion: we might have a better error message if we print out the `s` array to give a person more context to where the error is, particularly if there are many create units lists.